iOS-Email-Client/Libs/Env.swift

```diff
@@ -49,7 +49,7 @@ struct Env {
         guard !Env.isProduction else {
             return "wss://socket.criptext.com"
         }
-        return "wss://socket.criptext.com"
+        return "wss://stagesocket.criptext.com"
```

Comment:
PR should not have staging urls
```suggestion
        return "wss://socket.criptext.com"
```